zql/docs/aggregate-functions/README.md

```diff
@@ -66,6 +70,49 @@ QUICKEST LONGEST     TYPICAL
 
 # Available Aggregate Functions
 
+## `and`
+
+|                           |                                                                |
+| ------------------------- | -------------------------------------------------------------- |
+| **Description**           | Returns the boolean value `true` if the provided expression evaluates to `true` for all input values. Contrast with [`or`](#or). |
```

Comment:
```suggestion
| **Description**           | Returns the boolean value `true` if the provided expression evaluates to `true` for all input records. Contrast with [`or`](#or). |
```

Reply:
@henridf: Is this proposed change of wording a reflection of personal preference, or is there a substantial correctness argument here that I need to better understand? My use of the word "values" came from the same place as how I worded the high-level definition of an aggregate function at the top of the same doc (and therefore I'm prepared to accept critiques of this as well):

> A pipeline may contain one or more aggregate functions, which operate on batches of events to carry out a running computation over values contained in the events.

So now here when documenting `and()`, I was using "values" as shorthand for something like "what the expression evaluates to based on the values it takes as inputs and the computations performed on them in that expression". To me, using "records" here instead implies the aggregate function is somehow operating on the entirety of an event rather than just the parts referenced in the expression.

Having written that all out, now I'm wondering if I should just say "...for all inputs" and leave it at that. What do you think?

Reply:
> @henridf: Is this proposed change of wording a reflection of personal preference, or is there a substantial correctness argument here that I need to better understand?

Since the expression is evaluated over an input record (matching the sentence you quoted, `operate on batches of events`, modulo the use of event vs record), it seemed more accurate to phrase it this way.  

I like your proposed "for all inputs". It's shorter, at least as clear, and as you've pointed out elsewhere, this is trying to be a helpful doc rather than a formal spec. 👍

Reply:
Thanks @henridf! "...for all inputs" it is.